### PR TITLE
BugFix for tests with warnings

### PR DESCRIPTION
**Pull request by Dimundra**
dev
## CLICKUP

* [CLICKUP task](https://app.clickup.com/t/kh6na8)

## Code reviewers
@ostannja 
@Ihor-Kozachok 
@TionCada 

## Summary of issue

All warnings eradicated, code refactored, coverage remained at the same level

## Summary of change

Fixed 7 testing files

## Testing approach

testing-library

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by Igor-Kozachok — APPROVED** [review]

File: src/components/banner/banner.test.js [hunk 1]
```diff
@@ -36,30 +36,29 @@ describe('banner elements', () => {
 });
 
 describe('banner events', () => {
-  test('should change image position on mousemove once every 100ms', () => {
+  test('should change image position on mousemove once every 100ms', async () => {
     const { getByAltText } = render(<Banner />);
     const targetElem = getByAltText('img');
+    jest.useFakeTimers();
```
Comment (Dimundra) [inline src/components/banner/banner.test.js:42]: investigate this line

File: src/components/banner/banner.test.js [hunk 2]
```diff
@@ -36,30 +36,29 @@ describe('banner elements', () => {
...
-  test('should fire scroll event once button is clicked', () => {
+  test('should fire scroll event once button is clicked', async () => {
     const handleClick = jest.fn();
     const { getByRole } = render(<Banner handleClick={handleClick} />);
     const button = getByRole('button');
 
     fireEvent.click(button);
-
-    expect(handleClick).toHaveBeenCalled();
+    await wait(() => {
```
Comment (Dimundra) [inline src/components/banner/banner.test.js]: investigate if need to remove wait

File: src/components/forCabinetPage/map/map.test.js [hunk 3]
```diff
@@ -19,7 +19,7 @@ import InstitutionsOfEducationMap from './index';
 // });
```
Comment (Dimundra) [inline src/components/forCabinetPage/map/map.test.js]: Add more functionality(new task)

File: src/components/newPasswordForm/newPasswordForm.test.js [hunk 4]
```diff
@@ -30,6 +30,10 @@ jest.mock('react-router-dom', () => ({
 
 jest.useFakeTimers();
 
+afterEach(() => {
```
Comment (Dimundra) [inline src/components/newPasswordForm/newPasswordForm.test.js]: move to describe block

File: src/components/superAdmin/addInstitutionOfEducationForm/addInstitutionOfEducationForm.test.js [hunk 5]
```diff
@@ -1,18 +1,16 @@
...
 import userEvent from '@testing-library/user-event';
 import { Provider } from 'react-redux';
 import { store } from '../../../store/store';
 import { BrowserRouter as Router } from 'react-router-dom';
 import { authContext } from '../../../services/tokenValidator';
 
-jest.setTimeout(15000)
+jest.setTimeout(15000);
+
+window.scrollTo = jest.fn();
```
Comment (Dimundra) [inline src/components/superAdmin/addInstitutionOfEducationForm/addInstitutionOfEducationForm.test.js]: move to describe block, or put into beforeEach

File: src/components/banner/banner.test.js (hunk 1 above)
Comment (Dimundra) [reply src/components/banner/banner.test.js:42]: If remove this line it will cause new warning to appear on npm run test:coverage, so it should stay

**Review by ostannja — APPROVED** [review]
Approved.

**Review by TionCada — APPROVED** [review]

**Pull request merged**